gulpfile.js

```diff
@@ -51,77 +61,135 @@ gulp.task('serve', function (done) {
   });
 });
 
+
+/**
+ * Serve the Harp Site in production
+ */
+gulp.task('serveprod', ['build'], function() {
+  harp.server('.', {
+    port: process.env.PORT || 5000
+  }, function () {
+  });
+});
+
+
 /**
  * Build the Harp Site
  */
-gulp.task('build', ['jimp', 'uglify'], function (done) {
-  cp.exec('harp compile . www', {stdio: 'inherit'})
-    .on('close', done);
+gulp.task('build', ['copy-css-assets', 'jimp', 'uglify'], function () {
+  return cp.exec('harp compile . www', {stdio: 'inherit'})
+});
+
+
+// Jimp variables
+var imgSrc          = 'public/_posts-images/*',
+    imgDest         = 'public/images/posts/',
+    imgQuality      = 80,
+    largeWidth      = harpConfig.globals.breakpoints.large,
+    regularWidth    = harpConfig.globals.breakpoints.regular,
+    mediumWidth     = harpConfig.globals.breakpoints.medium,
+    smallWidth      = harpConfig.globals.breakpoints.small
```

Comment:
We should not introduce extra spaces (see var declarations up top).

We also need to refactor this a bit. We cannot be declaring variables in multiple places. JS code standards say that you declare all the variables you're going to use at the top of your function - first line. At the very least we need to move this to the top of the file but the best thing would be to only declare each one at the top of the functions it is used in. Alternatively, we could try to wrap this whole thing in a function and put these variables there.